site/docs/patterns/international-address-input.mdx

```diff
@@ -145,6 +144,6 @@ You can add a switch to toggle localization after the country selection, enablin
   />
 </Diagrams>
 
-## Expanding the pattern
+## Feedback
```

Comment:
Is this going to be standard across all the pages? If so we should just create a component to template it.